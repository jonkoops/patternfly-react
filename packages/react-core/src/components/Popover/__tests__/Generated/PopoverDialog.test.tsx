/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { PopoverDialog } from '../../PopoverDialog';

it('PopoverDialog should match snapshot (auto-generated)', () => {
  const { asFragment } = render(<PopoverDialog position={'top'} className={'null'} children={<>ReactNode</>} />);
  expect(asFragment()).toMatchSnapshot();
});
