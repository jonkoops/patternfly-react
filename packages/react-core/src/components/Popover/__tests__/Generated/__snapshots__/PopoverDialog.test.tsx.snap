// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PopoverDialog should match snapshot (auto-generated) 1`] = `
<DocumentFragment>
  <div
    aria-modal="true"
    class="pf-v5-c-popover pf-m-top null"
    role="dialog"
  >
    ReactNode
  </div>
</DocumentFragment>
`;
